manifest_staging/charts/gatekeeper/values.yaml

```diff
@@ -1,4 +1,4 @@
-replicas: 3
+replicas: 1
```

Comment:
This doesn't match what is in `cmd/build/helmify/static/values.yaml`, which it should if you ran `make manifests` before committing.

Reply:
Hi there.
Thanks for your review. This PR is in working in progress, and it required more discussion for avoid
all the back and forth.
From other notes regarding parameter naming it come from rational that user need to inform that 
mutation capability at the moment of writing is in prealpha status.

For more detail about why that label was choose please checkout 
https://zoom.us/rec/play/S01t6eZ3rvmCTVHezSsfT87S4i58v1Uw2IZZ2yg835pvOYg9GowoHAMng2re7BtH3ryam3W72p9qVqnK.v5SMnfBosNaKkPkw?startTime=1613581252000&_x_zm_rtaid=ttIPoIxPQqerbOeYe-N86w.1617260015803.dcd20ae36b99a9e7c67294644da12649&_x_zm_rhtaid=10 at 00:14:40.

this not mean that the label name is written on the stone ;-)